index.spec.js

```diff
@@ -3,30 +3,59 @@ const sinon = require('sinon');
 const logger = require('.');
 
 describe('ExceptOnTestLogger', () => {
-  let consoleLogStub;
+  let consoleLogSpy;
+  let consoleErrorSpy;
+  let consoleInfoSpy;
+  let consoleWarnSpy;
 
   beforeEach(() => {
-    consoleLogStub = sinon.stub(console, 'log');
+    consoleLogSpy = sinon.spy(console, 'log');
+    consoleErrorSpy = sinon.spy(console, 'error');
+    consoleInfoSpy = sinon.spy(console, 'info');
+    consoleWarnSpy = sinon.spy(console, 'warn');
   });
 
   afterEach(() => {
-    consoleLogStub.restore();
+    consoleLogSpy.restore();
+    consoleErrorSpy.restore();
+    consoleInfoSpy.restore();
+    consoleWarnSpy.restore();
   });
 
   describe('.log()', () => {
     context('when environment is "test"', () => {
       it('does not invoke console.log', () => {
         process.env.NODE_ENV = 'test';
         logger.log('Hello world.');
-        sinon.assert.notCalled(consoleLogStub);
+        sinon.assert.notCalled( consoleLogSpy );
```

Comment:
For consistency can you revert these formatting changes? The other functions don't have spaces inside the parenthesis. example the describe and context blocks, sinon.spy calls etc.